src/UserCollectableAttribute.js

```diff
@@ -112,6 +114,21 @@ class UserCollectableAttribute {
     return this;
   }
 
+  initializeValuesWithArrayItems(identifier, values, version) {
+    const definition = UserCollectableAttribute.getDefinition(identifier, version, this.definitions);
+
+    const ucaArray = [];
+
+    if (!_.isArray(values)) throw new Error(`Value for ${identifier}-${version} should be an array`);
+
+    _.forEach(values, (value) => {
```

Comment:
wouldn't this be nicer with a one-liner map rather than a forEach?
i.e. 
`
const ucaArray = values.map(value => new UserCollectableAttribute(_.get(definition, 'items.type'), value));
`